src/lib/dexrt.cpp

```diff
@@ -51,6 +51,12 @@ int64_t dex_allocation_size (char* ptr) {
   return *(reinterpret_cast<int64_t*>(ptr - alignment));
 }
 
+void* dex_pthread_key_create () {
+  pthread_key_t* key_ptr = (pthread_key_t*) malloc(sizeof(pthread_key_t));
+  pthread_key_create(key_ptr, NULL);
```

Comment:
You could use the destructor to free the key pointer. It leaks memory right now.

Reply:
As discussed, it's a tiny, fixed amount of memory once per process, so we don't need to worry about destructors too much and it's unclear when we'd call them anyway, except for at program termination. I'll add a comment to remind us that we're being a bit naughty.